### ISSUE TITLE
Make job_stats non-core

### ISSUE THREAD

**Issue by joehandzik** [HewlettPackard/lustre_exporter#142]
As suggested in #138

**Comment by sknolin** [comment]
I've been working with this for a bit now with job_stats as non-core. It's easy, but I don't think this is a good answer for the long term, because you may want extended stats but job_stats.

I think it needs a flag collector.jobstats - this does complicate the code.

**Comment by joehandzik** [comment]
Yeah, that's really what I'd intend to do, make a separate jobstats flag. Just a matter of time and priorities, there were two of us who built this code on the HPE side and we're all way underwater on other projects. I may have cycles next week to get this done (finally).
